docs/xcm/integration/add-to-portal.md

```diff
@@ -0,0 +1,96 @@
+---
+sidebar_position: 5
+---
+
+# Add assets into Astar Portal
+
+Once a channel is opened between parachains and assets have been registered, developers can create a PR to [our portal](https://github.com/AstarNetwork/astar-apps) for adding assets to our portal. Here’s what you need to know about the integration of the XCM assets into our portal.
+
+## Define the parachains information
+
+1. Add the `Chain name` at the [Chain](https://github.com/AstarNetwork/astar-apps/blob/main/src/v2/models/XcmModels.ts#L2) enum.
+2. Add the `parachain id` at the [parachainIds](https://github.com/AstarNetwork/astar-apps/blob/main/src/v2/models/XcmModels.ts#L2) enum.
```

Comment:
minor typo - should link to https://github.com/AstarNetwork/astar-apps/blob/main/src/v2/models/XcmModels.ts#L17

Reply:
I've fixed in [this](https://github.com/AstarNetwork/astar-docs/pull/91/commits/b7859f20a3b5454774885619a722fe287184b438) commit. Thank you!